conf.yaml

```diff
@@ -2397,7 +2421,6 @@ contents:
             current:    6.2
             subject:    X-Pack
             index:      docs/en/index.asciidoc
-            private:    1
```

Comment:
> private: 1

I suspect these "private" settings were used to omit "edit URLs" from these legacy docs.  Is this change necessary?

Reply:
`private` is defined twice for these books (see line 2418 in the new version or 2394 in the old version). I've been meaning to clean this up forever (VS Code reports the problem when I'm editing the conf.yaml file)